### ISSUE TITLE
SyntaxError: Unexpected end of JSON input

### ISSUE THREAD

**Issue by ArturiaPendragon** [DasKraken/CR-dl#51]
Hi, I ran this command line

`cr-dl dl "https://www.crunchyroll.com/fr/sing-yesterday-for-me/episode-3-what-is-love-794608"`

but ...

`SyntaxError: Unexpected end of JSON input
    at JSON.parse (<anonymous>)
    at MediaVilosPlayer.getSeriesTitle (/usr/lib/node_modules/cr-dl/src/MediaVilosPlayer.js:145:53)
    at Object.downloadVideoUrl (/usr/lib/node_modules/cr-dl/src/CrDl.js:554:38)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
    at async run (/usr/lib/node_modules/cr-dl/index.js:245:7)`

Thanks in advance !

**Comment by WawitoCL** [comment]
I have the same problem :( 

```
cr-dl download https://www.crunchyroll.com/sing-yesterday-for-me
...
... SING &quot;YESTERDAY&quot; FOR ME Episodio 1
...
...
SyntaxError: Unexpected end of JSON input
    at JSON.parse (<anonymous>)
    at MediaVilosPlayer.getSeriesTitle (/usr/local/lib/node_modules/cr-dl/src/MediaVilosPlayer.js:145:53)
    at Object.downloadVideoUrl (/usr/local/lib/node_modules/cr-dl/src/CrDl.js:554:38)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
    at async run (/usr/local/lib/node_modules/cr-dl/index.js:245:7)
```

Thanks in advance!

**Comment by DasKraken** [comment]
Yeah thats a bug. That " in title confuses the program.

**Comment by chichid95** [comment]
Hello, 
I think I have the same thing, and for the same series: D
Thanks for your tool !!!!
![image](https://user-images.githubusercontent.com/33454664/80873217-3556b480-8cb7-11ea-9188-877b872db109.png)
[](url)

**Comment by DasKraken** [comment]
Fixed in 4.0.0
